static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -663,34 +668,36 @@ export class AccessMethod extends Component {
                       nested
                     >
                       {
-                        selectedVariables.length > 0 && (
-                          <p className="access-method__section-status">
-                            {`${selectedVariables.length} ${pluralize('variable', selectedVariables.length)} selected`}
-                          </p>
-                        )
-                      }
-
-                      {
-                        selectedVariables.length === 0 && (
+                        !hasVariables ? (
                           <p className="access-method__section-status">
-                            No variables selected. All variables will be included in download.
+                            This service has no associated variables.
```

Comment:
I'm hesitant with the verbiage of the "service" has not associated variables just because of the fact that for most items its actually the collection which is associated to the service. That said I am not sure of the best way to phrase this. I think something like: `This item has not associated variables` might be okay. Approving based on the assumption we can just work out a better phrase or our team is okay with the current verbiage and I am just taking it too literally

Reply:
Yeah I don't love the wording for that reason as well, but I couldn't come up with anything that wasn't super verbose and even more confusing. I'm open to suggestions if anyone else has any though.